jekyll/_cci2/config-policy-reference.adoc

```diff
@@ -0,0 +1,275 @@
+---
+description: Open preview of config policy management for CircleCI
+version:
+- Cloud
+---
+= Config policy reference
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+NOTE: Config policy management is available on the **Scale** plan and is currently in **open preview**. All aspects of the feature are subject to change.
+
+The `circle-policy-agent` package includes built-in functions for common config policy
```

Comment:
@davidmparrott @sagar-connect I'm not clear on what this package is - sounds like the helpers listed on this page are in this package? But they are all in `package org`?

Reply:
No the helper functions live in a package called `circleci.config` (helpers for the config management)

And can be accessed via an import:

```
import data.circleci.config

config.SomeHelper
```

Reply:
So what is "The `circle-policy-agent` package"? is this correct? @davidmdm